src/Components/ProductsList.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+class ProdutctsList extends React.Component {
+  render() {
+    return (
+      <div>
+        <input />
+        <p data-testid="home-initial-message">
+          Digite algum termo de pesquisa ou escolha uma categoria.
+        </p>
+      </div>
+    );
+  }
+}
+
+export default ProdutctsList;
```

Comment:
@Marc-Web, tranquilo?

Vi um erro aqui no seu código, vc acabou escrevendo 'ProdutctsList' e no App.js vc chamou 'ProductsList'.

Mas queria fazer uma **sugestão**. Em vez de usar o "ProductsList", que tal vc mudar para "ProductsSearchBar" ou "SearchBar"? Pois fica mais inteligível o que o componente faz. O que vc acha? 😄

Caso mudar, só não esquece de tbm mudar no App.js 😃 
```suggestion
import React from 'react';

class ProductsSearchBar extends React.Component {
  render() {
    return (
      <div>
        <input />
        <p data-testid="home-initial-message">
          Digite algum termo de pesquisa ou escolha uma categoria.
        </p>
      </div>
    );
  }
}

export default ProductsSearchBar;
```

Reply:
Obrigado pela sugestão, faz todo sentido e vou implementá-la no meu código. VQV!!